src/TourManager/Travely.TourManager.API/Startup.cs

```diff
@@ -0,0 +1,83 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.HttpsPolicy;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Travely.TourManager.BLL;
+using Travely.TourManager.Core;
+using Travely.TourManager.DAL;
+using Newtonsoft.Json;
+
+namespace Travely.TourManager.API
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+
+            services.AddDbContext<DataContext>(x =>
+            {
+                x.UseSqlServer(Configuration.GetConnectionString("DefaultConnection"));
+            });
+
+            services.AddMvc().AddNewtonsoftJson();
+
+            services.AddScoped<ITourTypeService, TourTypeService>();
+            services.AddScoped<ITourService, TourService>();
+            services.AddScoped<ITourStatusService, TourStatusService>();
+            services.AddScoped<ILanguageService, LanguageService>();
+            services.AddScoped<IGenderService, GenderService>();
```

Comment:
Please, create `ServiceCollectionExtensions` in the services layer and move injection logic there